tests/subsys/bluetooth/mesh/sensor_subsys_new/src/main.c

```diff
@@ -0,0 +1,984 @@
+/*
```

Comment:
I really like the new testing approach in preference to the previous one. Good design work.